lemur/auth/service.py

```diff
@@ -104,10 +109,20 @@ def decorated_function(*args, **kwargs):
             header_data = fetch_token_header(token)
             payload = jwt.decode(token, current_app.config["LEMUR_TOKEN_SECRET"], algorithms=[header_data["alg"]])
         except jwt.DecodeError:
+            try:
+                log.info("DecodeError, will try treating token as a dd id_token")
+                data = JWTAuthenticator.instance("lemur_vault_authenticator").authenticate(token)
+                log.info("DD Authenticated Token Data: " + str(data))
```

Comment:
Assuming this doesn't log any sensitive information, let's drop the DD specific bits:
```suggestion
                log.info("Authenticated Token Data: " + str(data))
```